### PR TITLE
Prodect details page

### PR DESCRIPTION
**Pull request by Levi-Ojukwu**
Product details page

### CODE REVIEW

**Comment by ToksNet** [comment]
product details page pulled from levi work

**Review by ghost — COMMENTED** [review]
new change

File: src/components/NavBar.jsx [hunk 1]
```diff
@@ -181,37 +183,60 @@ const NavBar = () => {
...
+            </div>
+          </div>
+
+          <div className="flex flex-col justify-between">
+            <div className="md:border-r-[1px] pl-16 pr-20 flex flex-col border-[#D4B4B4] p-4">
+              <Link to="/home" className="flex justify-between items-center mb-[10px]">
+                <span>Women's Fashion</span>
+                <FaAngleRight className="w-4 h-4" />
+              </Link>
+              <Link to="/home" className="flex justify-between items-center mb-[10px]">
```
Comment (ToksNet) [inline src/components/NavBar.jsx:201]: removed gap 11

File: src/components/ProductDescripComponents/FullProductCard.jsx [hunk 2]
```diff
@@ -1,271 +1,271 @@
-import React, { useState } from "react";
-import { Prod1 } from "../../constants/constants";
-import Star from "../Star";
-import { FaHeart } from "react-icons/fa";
+// import React, { useState } from "react";
```
Comment (ToksNet) [inline src/components/ProductDescripComponents/FullProductCard.jsx:2]: commented out

**Pull request merged**